test/extensions/filters/http/ext_proc/test_attribute_builder.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include "source/extensions/filters/common/expr/evaluator.h"
+#include "source/extensions/filters/http/ext_proc/attribute_builder/attribute_builder.h"
+#include "source/extensions/filters/http/ext_proc/matching_utils.h"
+
+#include "test/extensions/filters/http/ext_proc/test_attribute_builder.pb.h"
+#include "test/extensions/filters/http/ext_proc/test_attribute_builder.pb.validate.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace ExternalProcessing {
+
+class TestAttributeBuilder : public AttributeBuilder {
+public:
+  using TestAttributeBuilderConfig =
```

Comment:
nit: two alias with the same type alias in header file. fully qualified names is fine. 
we could add the using alias in the cc file with fully qualified name "::xxx::...".